.github/workflows/build.yml

```diff
@@ -28,9 +29,94 @@ jobs:
         terraform init -backend=false
         terraform validate -no-color
       working-directory: terraform
-
+  
```

Comment:
is this whitespace char needed?

Reply:
@merrilmathew you may want to enable this option in VScode:
- Trim trailing whitespace